fbw-a32nx/src/wasm/fbw_a320/src/FlightDataRecorder.cpp

```diff
@@ -51,6 +54,38 @@ void FlightDataRecorder::update(AutopilotStateMachine* autopilotStateMachine,
   manageFlightDataRecorderFiles();
 
   // write data to file
+
+  // ELACs
+  for (int i = 0; i < 2; ++i) {
+    auto bus_outputs = elacs[i].getBusOutputs();
+    fileStream->write((char*)(&bus_outputs), sizeof(bus_outputs));
```

Comment:
These casts are undefined behaviour (strict aliasing rule).

Reply:
Do you have an idea how to improve this part of the code?

Reply:
Disregard, you actually can do this safely with char specifically so you're all good.